rados/object_iter.go

```diff
@@ -0,0 +1,94 @@
+package rados
+
+// #cgo LDFLAGS: -lrados
+// #include <errno.h>
+// #include <stdlib.h>
```

Comment:
errno.h and stdlib.h do not seem to be used?

Reply:
stdlib is needed for free afaict. errno can be removed.